README.md

```diff
@@ -34,3 +34,23 @@ A mod which enables Hollow Knight to act as an Archipelago client, enabling mult
 4. Enter in the correct settings for your Archipelago server.
 5. Hit **Start** to begin the game. The game will stall for a few seconds while it does all item placements.
 6. The game will immediately drop you into the randomized game. So if you are waiting for a countdown then wait for it to lapse before hitting Start, or hit Start then pause the game once you're in it.
+
+# Contributing
+Contributions are welcome, all code is licensed under MIT license. Please track your work within the repository if you are taking on a feature. This is done via GitHub Issues. If you are taking on an issue please comment on the issue and assign yourself (if possible). If you are looking to contribute something that isn't in the issues list then please submit an issue to describe what work you intend to take on.
```

Comment:
Perhaps insert "the MIT License", inserting the and capitalizing License as its a proper name for it.

Reply:
Coming in a following PR.